api/app/main.py

```diff
@@ -80,6 +81,16 @@
 
 ORIGINS = config.get('ORIGINS')
 
+
+async def catch_exception_middleware(request: Request, call_next):
+    try:
+        return await call_next(request)
+    except Exception as exc:
+        logger.debug(exc, exc_info=True)
```

Comment:
Log it as error please - not debug.
```suggestion
        logger.error(exc, exc_info=True)
```

Reply:
@Sybrand shouldn't we be posting this to the RC channel?

Reply:
Any unhandled exception that bubbles up to this level seems quite severe

Reply:
At this point going to rocket chat server no purpose, given the sheer amount of garbage logged there - it's essentially useless.
That's a separate concern though. I agree - logging to rocket chat would be a good idea.

Reply:
I agree that RC is noisy but I think it's better than log output that disappears with ephemeral pods. I still use RC messages to troubleshoot and be alerted to issues.

Reply:
Also, won't a raised error here cause the pod to shutdown anyways, making this log useless?